desc/objectives/_geometry.py

```diff
@@ -698,18 +728,33 @@ def build(self, use_jit=True, verbose=1):
             "quad_weights": w,
         }
 
+        if self._use_signed_distance:
+            # get the indices corresponding to the grid points
+            # at each distinct zeta plane, so that can be used
+            # in compute to separate the computed pts by zeta plane
+            zetas = plasma_grid.nodes[plasma_grid.unique_zeta_idx, 2]
+            plasma_zeta_indices = [
```

Comment:
isnt this basically `grid.inverse_zeta_idx`?

Reply:
Yeah I don't think we need all this logic because it is already done in the `Grid` class (and we already checked the grids have the same zeta values)